opm/simulators/wells/WellState.cpp

```diff
@@ -273,6 +273,7 @@ void WellState<Scalar>::init(const std::vector<Scalar>& cellPressures,
                                                                 report_step,
                                                                 wells_ecl);
     well_rates.clear();
+    this->inactive_well_names_ = schedule.getInactiveWellNamesAtEnd();
```

Comment:
If it is using `schedule.getInactiveWellNamesAtEnd();` to determine whether a well can be split across the processes, I will suggest a more specific name for `inactive_well_names_`,  to show the wells will be shut all the time and can not be open across the simulation.  Like `permanently_inactive_well_names_` and the corresponding function name can be `is_permanently_inactive_well`.